misk-web/web/packages/@misk/core/src/ducks/simpleDucksUtilities.ts

```diff
@@ -0,0 +1,131 @@
+/**
+ * simple*Ducks libraries common code
+ */
+
+/**
+ * Handler Functions
+ * Reduce the legwork of parsing `onChange`, `onClick` and other events in Buttons, Inputs, Toggles...etc to call your handling function
+ *
+ * Old Way: Manually declare inline arrow funtions consuming the input events
+ * ```
+ * <InputGroup onChange={(event: any) => (props.simpleFormInput("FormInputTag", event.target.value))}
+ * <NumberInput onChange={({valueAsNumber: number, valueAsString: string}) => (props.simpleFormNumber("FormNumberTag", valueAsNumber, valueAsString))}
+ * ```
+ *
+ * New way: Use on*FnCall to wrap and implicitly pass into handling functions the input events
+ * ```
+ * <InputGroup onChange={onChangeFnCall(props.simpleFormInput, ["FormInputTag"])}
+ * <NumberInput onChange={onChangeNumberFnCall(props.simpleFormNumber, ["FormNumberTag"])}
```

Comment:
my two cents: this doesn't really seem like that much less "legwork", especially at the cost of adding another set of utility APIs that developers will have to understand to use when building on misk-web. Of course, I don't have full context here :-).

Reply:
I'm certainly open to removing this if it doesn't make it easier for devs.